azure-pipelines.yml

```diff
@@ -7,11 +7,11 @@ schedules:
     displayName: Nightly build
     branches:
       include:
-        - master
+        - branch-5.15
```

Comment:
We should have `branch-*` on master too, for the future (this one is fine here as is)

Reply:
..hmm.. will this one with the trigger be even picked up from a non-master branch? I don't now

Reply:
I've added https://github.com/SonarSource/sonar-scanner-msbuild/pull/1897